4/main.js

```diff
@@ -0,0 +1,123 @@
+const url = 'https://opentdb.com/api.php?amount=10&type=multiple';
+const title = document.getElementById ('title');
+const quizLevel = document.getElementById ('quizLevel');
+const text = document.getElementById ('text');
+const startBtn = document.getElementById ('btn');
+const btnArea = document.getElementById ('btnArea');
+let quizList;
+let curIndex = 0;
+let correctAnswers = 0;
+
+btn.addEventListener ('click', getQuizList);
```

Comment:
メソッドと引数のカッコ `()` の間に半角スペースが入っているようですが
こちらはフォーマットの設定によるものでしょうか？
私個人はあまりこのようにフォーマットされたコードを見たことがなかったので確認です。

```suggestion
btn.addEventListener('click', getQuizList);
```